docs/book/installation.md

```diff
@@ -0,0 +1,5 @@
+**This is only a placeholder.**
+
+The content of this page can be found under:
+
+https://github.com/laminas/documentation-theme/blob/master/theme/pages/installation.html
```

Comment:
I'm sorry I don't understand: is this PR ready to be merged?

Reply:
Yes! This _placeholder file_ itself is needed for MkDocs otherwise nothing is rendered.